WordPress/WordPressShareExtension/NSExtensionContext+Extensions.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// Encapsulates NSExtensionContext Helper Methods.
+///
+extension NSExtensionContext {
+    
+    func loadWebsiteUrl(completion: (NSURL? -> Void)) {
+        let publicUrlIdentifier = "public.url"
+        
+        guard let item = inputItems.first as? NSExtensionItem,
+            let itemProviders = item.attachments as? [NSItemProvider] else
+        {
+            completion(nil)
+            return
+        }
+        
+        let urlItemProviders = itemProviders.filter { (itemProvider) in
+            return itemProvider.hasItemConformingToTypeIdentifier(publicUrlIdentifier)
+        }
+        
+        guard urlItemProviders.count > 0 else {
+            completion(nil)
+            return
+        }
+        
+        itemProviders.first!.loadItemForTypeIdentifier(publicUrlIdentifier, options: nil) { (url, error) in
```

Comment:
OMG this is clearly an error!! `urlItemProviders` isn't being referenced after filtering!. Nice catch!